docs/appendices/release-notes/unreleased.rst

```diff
@@ -58,6 +58,9 @@ None
 Fixes
 =====
 
+- Fixed an issue that could lead to stuck queries when joining more than
+  2 tables on the ``primary key`` column.
```

Comment:
```suggestion
- Fixed an issue that could lead to stuck queries when joining more than two
  relations and if the WHERE clause contained filters on primary key columns
  that led to an execution plan using a ``primary key`` lookup on at least one
  relation.
```

In the previous version it sounds as if the `ON` join clause had to happen on the primary key column which is misleading.